doc/user/sql-grammar/generate.sh

```diff
@@ -38,4 +38,11 @@ mkdir scratch
         fi
     done
 )
+
+rm -rf $dest
+mv scratch $dest
+
+# ping hugo again
+sleep 3
+mv $dest scratch
 mv scratch $dest
```

Comment:
But this part seems real sketchy. Does just touching a file in the directory not work?

Reply:
yeah touch $dest/* does also work.